app/scripts/controllers/transactions.js

```diff
@@ -78,6 +70,58 @@ module.exports = class TransactionController extends EventEmitter {
     return this.store.getState().transactions
   }
 
+  get unapprovedTxCount () {
```

Comment:
maybe just use a function
i think its clearer that its doing some calc and could have side effects when its a fn
by making a getter we're sort of pretending its a property